src/common/modules/data/Tips.js

```diff
@@ -103,12 +103,12 @@ const tipArray = [
         },
         randomizeDisplay: false,
         text: "tipYouLikeAddon",
-        actionButton: {
+            actionButton: {
```

Comment:
```suggestion
        actionButton: {
```

arg wrong intendation